src/@chakra-ui/gatsby-plugin/components/Modal.ts

```diff
@@ -11,57 +11,67 @@ const {
   defaultProps,
 } = modalDefaultTheme
 
-const variantCode = definePartsStyle((props) => ({
-  overlay: {
-    bg: "rgba(0, 0, 0, 0.7)",
-  },
-  dialog: {
-    maxW: "100vw",
-    marginTop: "auto",
-    marginBottom: 0,
-    maxHeight: "50%",
-    borderRadius: 0,
-  },
-  header: {
-    bg: props.colorMode === "dark" ? "rgb(25, 25, 25)" : "rgb(247, 247, 247)",
-    borderColor:
-      props.colorMode == "dark" ? "rgb(242, 242, 242)" : "rgb(51, 51, 51)",
-    borderTop: "1px solid",
-    borderBottom: "1px solid",
-    textTransform: "uppercase",
-    fontWeight: "normal",
-    fontSize: "md",
-    fontFamily: "monospace",
-  },
-  closeButton: {
-    padding: 0,
-    width: "24px",
-    height: "24px",
-    borderRadius: 0,
-    color: "rgb(178, 178, 178)",
-    fontSize: "sm",
-    margin: 0,
-    top: 4,
-    right: 4,
-    bottom: 4,
-  },
-  body: {
-    padding: 0,
-  },
-}))
+// Remove all the comment line once the review is completed & PR will able to merge
+
+/* const variantCode = definePartsStyle((props) => ({
+ 
+})) */
 
 export const Modal = defineMultiStyleConfig({
   baseStyle: definePartsStyle((props) =>
     defineMergeStyles(defaultBaseStyle?.(props), {
+      overlay: {
+        bg: "rgba(0, 0, 0, 0.7)",
+      },
       dialog: {
-        my: 0,
+        size: "2xl",
+        /* maxW: "464px", */
+        my: "64px",
+        mx: "8px",
+        gap: 4,
+        padding: "32px",
+        maxHeight: "70%",
+        borderRadius: "4px",
+      },
```

Comment:
```suggestion
        bg: "background.base",
        gap: "4",
        padding: "8",
        maxW: "464px",
        borderRadius: "md",
        boxShadow: "none",
      },
```
A couple of points here:
1. There should only be style props used in the theme objects, and `size` is not a style prop. The `maxW` prop should be used here because the default sizes to choose from use default size tokens and `464px` does not match any of the tokens. (@nloureiro for your awareness. The DS example has a width value that is not equal to any of the current token values to choose from. Need to consider a value of `448px` or `512px` if we are to use an existing token.)
2. Need to make sure we have the default `boxShadow` removed.
3. `borderRadius` needs to use the matching token value.
4. `maxHeight` does not need to be defined here, Unless @nloureiro has a particular height constraint in mind to force scrolling of the content.
5. The background color needs to be defined with a semantic token to ensure it matches with the DS (the default values do not match the DS)
6. Margin values do nothing here because the modal is absolutely positioned.